### PR TITLE
Updated comparative output to use toEqual instead of toBe

### PR DESCRIPTION
**Pull request by phillipzada**
See https://github.com/synapse-wireless-labs/jasmine-marbles/pull/4

### CODE REVIEW

File: index.ts [hunk 1]
```diff
@@ -1,34 +1,17 @@
...
-  TestColdObservable,
-  TestObservable,
-} from './src/test-observables';
-import {
-  initTestScheduler,
-  getTestScheduler,
-  resetTestScheduler,
-} from './src/scheduler';
-
-export {
```
Comment (brandonroberts) [inline index.ts]: These lines should not be deleted. The integration tests fail without them.

Comment (brandonroberts) [reply index.ts]: These lines are missing

```
export {
   getTestScheduler,
   initTestScheduler,
   resetTestScheduler,
 } from './src/scheduler';
```

**Comment by brandonroberts** [comment]
I like the new functionality this PR provides, but favor the original output message for an error.

**Comment by PhillipZada** [comment]
@brandonroberts the original output message using toBe was hard to decode if you marbles timings were off? @MikeRyanDev what are your thoughts?

**Comment by brandonroberts** [comment]
@phillipzada to me its a little bit more cryptic now but maybe its just because I'm not used it. If you will fix that export issue, I'll go ahead and merge it

**Comment by phillipzada** [comment]
Done

**Pull request merged**